Jellyfin.Api/Controllers/DashboardController.cs

```diff
@@ -45,6 +46,7 @@ public class DashboardController : BaseJellyfinApiController
     /// <response code="404">Server still loading.</response>
     /// <returns>An <see cref="IEnumerable{ConfigurationPageInfo}"/> with infos about the plugins.</returns>
     [HttpGet("web/ConfigurationPages")]
+    [Authorize(Policy = Policies.RequiresElevation)]
     [ProducesResponseType(StatusCodes.Status200OK)]
     [ProducesResponseType(StatusCodes.Status404NotFound)]
     [Authorize]
```

Comment:
I'm not familiar with the C# annotation function annotations, but this line looks redundant at a very least?

Reply:
Good call, it doesn't hurt adding it twice but it is not necessary. Will fix.